File: scratch/fix_tailwind.js
const fs = require('fs');

let html = fs.readFileSync('Views/Shared/_Layout.cshtml', 'utf8');

// Change tailwind config
html = html.replace("darkMode: 'class',", "darkMode: ['class', '[data-theme=\"dark\"]'],");

fs.writeFileSync('Views/Shared/_Layout.cshtml', html);
console.log('Fixed config');

File: scratch/fix_tailwind.py
import re
import io

def process_file():
    with io.open('Views/Shared/_Layout.cshtml', 'r', encoding='utf-8') as f:
        html = f.read()

    # Change tailwind config
    html = html.replace("darkMode: 'class',", "darkMode: ['class', '[data-theme=\"dark\"]'],")

    with io.open('Views/Shared/_Layout.cshtml', 'w', encoding='utf-8') as f:
        f.write(html)

if __name__ == '__main__':
    process_file()
